components/patient-access-rules/api/src/test/java/org/phenotips/data/permissions/internal/DefaultCollaboratorTest.java

```diff
@@ -167,6 +167,32 @@ public void equalsTest() throws ComponentLookupException
         Assert.assertFalse(c.equals("other"));
     }
 
+    /** Basic tests for {@link Collaborator#equals(Object)}. */
+    @Test
+    public void equalsWithNullUserTest() throws ComponentLookupException
+    {
+        Collaborator c = new DefaultCollaborator(null, this.access, this.helper);
+        // Equals itself
+        Assert.assertTrue(c.equals(c));
+        // Doesn't equal null
+        Assert.assertFalse(c.equals(null));
+        // Equals an identical collaborator
+        AccessLevel otherAccess = mock(AccessLevel.class);
```

Comment:
This isn't used until later, move this declaration on line 184.

Reply:
Fixed.